### ISSUE TITLE
Widgets

### ISSUE THREAD

**Issue by joern-vh** [MusicConnectionMachine/VisualizationG5#44]
Creating Widgets for [IMSLP](http://imslp.org/)
- Relationships
- Timeline (Component by @chaoran-chen)

- [ ] Clarify what Data we can get
- [ ] Thinking of how to inject the Widgets into IMSLP.  #43

**Comment by vviro** [comment]
@Binabiko @chaoran-chen What's the state of the IMSLP integration? Shouldn't this be re-opened or a new issue specifically concerning the integration of the widgets with the IMSLP be opened? I see this is covered in https://github.com/MusicConnectionMachine/VisualizationG5/pull/54, so probably the question belongs there, but do we have a demo of the IMSLP embedding? If so, we could test it on the weekend on the IMSLP proper.

**Comment by joern-vh** [comment]
What do you mean? IMSLP Support is already finished.

**Comment by chaoran-chen** [comment]
There is still one issue that needs to be done before we can really include them: https://github.com/MusicConnectionMachine/api/issues/101, but I believe that the front-end is ready.
